config/intellij-idea-inspections.xml

```diff
@@ -2151,6 +2153,9 @@ isolated classes and we cannot put them to separate package as it will affect us
                 <!-- XDocs UT is very long and complex -->
                 <option value="OverlyComplexBooleanExpression" />
                 <option value="ReuseOfLocalVariable" />
+                <!-- The output stream created by Main.createListener will be closed later. This is
+                       by design. A warning issued by Eclipse JDT need to be suppressed. -->
+                <option value="resource" />
```

Comment:
Please make comment more general. As we sometime postpone close for resource by design, not only for this particular case - Main.createListener

Reply:
sorry to enforce bureaucracy, but it is proven by practice, principle to do "done" for each point.
It is just final recheck that all items from numerous are done.